### PR TITLE
Switch to April Vision

### PR DESCRIPTION
**Pull request by WillB97**
Switches vision system from Zoloto to April Vision.
Also adds handling for vision backends not being defined in an environment.

Temporarily sets the package version to 2023.1.1+april1

### CODE REVIEW

**Comment by trickeydan** [comment]
I think this is probably dependent on review of the new library as well. Is there source published somewhere?

**Comment by WillB97** [comment]
The library source is at https://github.com/WillB97/april_vision

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "sr.robot3"
-version = "2023.1.0"
+version = "2023.1.1+april1"
```
Comment (trickeydan) [inline pyproject.toml]:
```suggestion
version = "2023.1.0"
```

We shouldn't be bumping this in a feature PR. Please make a separate version release PR.

File: sr/robot3/game.py [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-from typing import Container, Dict
+from typing import Dict, Iterable
 
 
 class UnusedMarkerException(Exception):
     """This marker is not used for the game."""
 
 
-MARKER_SIZES: Dict[Container[int], int] = {
+MARKER_SIZES: Dict[Iterable[int], int] = {
```
Comment (trickeydan) [inline sr/robot3/game.py:9]: This should be a Collection. We can address this later though as you mentioned you have mypy issues.

File: sr/robot3/vision/strategy.py [hunk 3]
```diff
@@ -69,3 +69,15 @@ def _find_default_camera(self) -> Optional[str]:
...
+# STRATEGIES: List[CalibrationStrategy] = [
+#     USBDevicePresentStrategy(0x046d, 0x0825, "Logitech C270"),
+#     USBDevicePresentStrategy(0x046d, 0x0807, "Logitech B500"),
+#     USBDevicePresentStrategy(0x046d, 0x080a, "Logitech C905"),
+#     USBDevicePresentStrategy(0x046d, 0x082d, "Logitech C920"),
+#     USBDevicePresentStrategy(0x046d, 0x0892, "Logitech C920"),
+#     USBDevicePresentStrategy(0x046d, 0x08E5, "Logitech C920"),  # C920 PRO
+#     USBDevicePresentStrategy(0x0c45, 0x6713, "Microdia Integrated_Webcam_HD"),
+#     MacSystemStrategy(),
+# ]
```
Comment (trickeydan) [inline sr/robot3/vision/strategy.py]:
```suggestion
```

**Review by trickeydan — CHANGES_REQUESTED** [review]
Nice! 🎉 

Please see inline comments and strip and calibration strategy code as discussed :)

**Review by trickeydan — APPROVED** [review]

**Comment by trickeydan** [comment]
Closes #74 
Closes #81

**Pull request merged**